src/nimblepkg/packageparser.nim

```diff
@@ -464,6 +464,7 @@ proc getInstalledPkgs*(libsDir: string, options: Options):
           pkg = readPackageInfo(nimbleFile, options, onlyMinimalInfo=false)
         except ValidationError:
           let exc = (ref ValidationError)(getCurrentException())
+          pkg = exc.pkgInfo
```

Comment:
By the way - this procedure (``getInstalledPkgs``) is not used anywhere, is that okay?